mr-clean-debug-processor/src/main/java/com/trello/mrclean/MrCleanDebugProcessor.kt

```diff
@@ -0,0 +1,29 @@
+package com.trello.mrclean
+
+import com.google.devtools.ksp.processing.*
+import com.google.devtools.ksp.symbol.*
+
+class MrCleanDebugProcessor(
+    val codeGenerator: CodeGenerator,
+    val options: Map<String, String>,
+    private val logger: KSPLogger,
+) : SymbolProcessor {
+    override fun process(resolver: Resolver): List<KSAnnotated> {
+        logger.info("Mr. Clean Debug Processor has $options")
```

Comment:
should we be retaining logger by default?

Reply:
I generally like leaving info logging around in case issues pop up while folks are running it, at least its a decent indicator if something goes sideways. More than once something breaking in a plugin has made me lose quite a bit of time chasing it down.

Could go either way though.